e2e/test/iothub/SasCredentialAuthenticationTests.cs

```diff
@@ -96,13 +94,21 @@ public async Task JobClient_Http_SasCredentialAuth_Success()
             try
             {
                 // act
-                JobResponse createJobResponse = await jobClient
+                ScheduledTwinUpdate twinUpdate = new ScheduledTwinUpdate
+                {
+                    QueryCondition = query,
+                    Twin = twin,
+                    StartTimeUtc = DateTime.UtcNow
+                };
+                ScheduledJobsOptions scheduledTwinUpdateOptions = new ScheduledJobsOptions
```

Comment:
var when the type is obvious (in this case we call `new ScheduledJobOptions` so it is obvious.

```suggestion
                var scheduledTwinUpdateOptions = new ScheduledJobsOptions
```